sarif/sarif.go

```diff
@@ -42,14 +42,45 @@ var typeName = map[int64]string{
 	200299: "Other",
 }
 
-func NewTransformer(scanResult *sarif.SchemaJson,
+func validateDataSource(dataSource *ocsffindinginfo.DataSource) error {
+	if dataSource == nil {
+		return errors.Errorf("no source provided for ")
```

Comment:
nit; no source provided for?

I can see further down
```
if err := validateDataSource(dataSource); err != nil {
		return nil, errors.Errorf("invalid data source provider: %w", err)
	}
```

Reply:
fixed